content/coronavirus_education_page.yml

```diff
@@ -68,11 +68,11 @@ content:
           list:
             - label: Exam cancellations, results and appeals
               url: /government/publications/coronavirus-covid-19-cancellation-of-gcses-as-and-a-levels-in-2020
-        - title: Managing a school or early years setting
+        - title: Managing a school or early-years setting
```

Comment:
This says you're adding the hyphen in - think this was left over from the last PR

Reply:
it was - not sure how to delete or edit that tbh

Reply:
let me try